core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -69,9 +69,16 @@
 /* Normal memory, Outer Write-Back Write-Allocate Cacheable */
 #define TEE_MMU_TTB_RNG_WBWA    (1 << 3)
 
+/* Normal memory, Outer Write-Back no Write-Allocate Cacheable */
+#define TEE_MMU_TTB_RNG_WB      (3 << 3)
+
+#ifndef CFG_NO_SMP
```

Comment:
Could we use `CFG_TEE_CORE_NB_CORE` instead or is it a valid use case to have one core, but still in SMP configuration?

Wouldn't `TTBCR`, `TCR_EL1` need to be update also?